app/javascript/utils/react-bootloader.jsx

```diff
@@ -24,27 +24,65 @@ Bugsnag.start({
 
 const ErrorBoundary = Bugsnag.getPlugin('react').createErrorBoundary(React)
 
+// This changes any extra things that need changing from the
+// turbo frame, such as body class or page title
+document.addEventListener('turbo:frame-render', (e) => {
+  const bodyClass = e.detail.fetchResponse.response.headers.get(
+    'exercism-body-class'
+  )
+  document.querySelector('body').className = bodyClass
+})
+
+/********************************************************/
+/** Add a loading cursor when a turbo-frame is loading **/
+/********************************************************/
+const setTurboStyle = (style) => {
+  const styleElemId = 'turbo-style'
```

Comment:
Prevent the `styleElemId` from being allocated each time the `setTurboStyle` function is called

```suggestion
const styleElemId = 'turbo-style'
const setTurboStyle = (style) => {  
```